### DISCUSSION TITLE
02 Creating a Simple App

### DISCUSSION THREAD

**Discussion by vikrampawar** [rumq/java-springboot-course#15] [General]
# 02 Creating a Simple App

Link to PPT: https://docs.google.com/presentation/d/1dRzG3eRyW2DAEgPZcUqeC-hp6hvnSVs4/edit?usp=share_link&ouid=105297118363914028052&rtpof=true&sd=true

**Comment by vikrampawar** [comment]
# PawarV

## Dec 1
I ran it . It runs successfully
I looked at `Demo02SimpleAppApplication` class.
This method x - made no sense to me.
I looked at PPT, I think X is Y.
##  Jan 2
I watched video today, finally X made sense

**Comment by adadedja** [comment]
**Dec 1**
I tried to run the application in VS code, but it was taking very long more than 5 min to run and then I opened the first demo `Demo02SimpleAppApplication` 
It ran successfully. 

I ran demo.restservices (Demo-11) and it ran successfully on port 8080 (http://localhost:8080/simple/productsV1) 
It will show an xml list of products because based on PPT 11 the JSON/XML serializers will convert Product objects to/from JSON or XML automatically, as appropriate.

_Demos from 1-5 were successfully ran during and after respective Spring Boot Classes_

Ran successfully demo-06 `configurationlocation` printed a Hello message from the bean component and the time set on the @Bean component.
